core/src/io/anuke/mindustry/ui/fragments/PlacementFragment.java

```diff
@@ -26,7 +26,7 @@
 public class PlacementFragment extends Fragment{
     final int rowWidth = 4;
 
-    public Category currentCategory = Category.distribution;
+    public Category currentCategory = Category.turret;
```

Comment:
Why change this?

Reply:
I saw that `rebuild()` below sets `currentCategory` to turret right away, and the initial category at the start of every game is turret. It's just needless nitpicking by me.